internal/controllers/device_config_controller.go

```diff
@@ -45,10 +47,17 @@ func NewDeviceConfigController(settings *config.Settings, logger *zerolog.Logger
 }
 
 type DeviceConfigResponse struct {
-	PidURL           string `json:"pidUrl"`
-	DeviceSettingURL string `json:"deviceSettingUrl"`
-	DbcURL           string `json:"dbcURL,omitempty"`
-	Version          string `json:"version"`
+	PidURL           string        `json:"pidUrl"`
+	DeviceSettingURL string        `json:"deviceSettingUrl"`
+	DbcURL           string        `json:"dbcURL,omitempty"`
+	Version          string        `json:"version"`
+	PendingJobs      []JobResponse `json:"pending_jobs"`
```

Comment:
Do we need `PendingJobs` here since Jobs will be in a separate api request from the device config?